tests/test_choose_disk.py

```diff
@@ -0,0 +1,151 @@
+import imp
+import inspect
+import library.choose_disk as choose_disk
+import mock
+import os
+from nose.tools import assert_equals, nottest
```

Comment:
I agree, I think ``py.test`` is the way to go here. From looking at the tests I think you might only need to change the ``assert`` lines.